modules/taunt/taunt.gd

```diff
@@ -1,13 +1,51 @@
 extends Node
 
 
+#region Internal Functions
 func _ready() -> void:
-	Modules.register_hook(Anticheat.request, anticheat)
+	Modules.register_hooks(handler)
```

Comment:
Maybe instead of passing a handler, we could pass `self`:

```suggestion
	Modules.register_hooks(self)
```

And then in `modules.gd`:
```gdscript
def register_hooks(module: Node):
    # ...
    module["%s_hook" % what.to_snake_case()].callv(info)
    # ...
```

Reply:
In fact, maybe every module should inherit some `Module` class that does this in `_ready` automatically. Then it would be less work and code for the individual modules.

Reply:
I think it would be better for users to specify themselves. It would also be different from how cards do it, since cards do something similar with abilities.